tests/parametric/test_config_consistency.py

```diff
@@ -73,7 +73,8 @@ def set_service_version_tags():
 @features.tracing_configuration_consistency
 class Test_Config_UnifiedServiceTagging:
     @parametrize("library_env", [{}])
-    def test_default_version(self, library_env, test_agent, test_library):
+    def test_default_config(self, library_env, test_agent, test_library):
+        assert library_env.get("DD_ENV") == None
```

Comment:
Technically we expect DD_ENV, DD_VERSION and DD_SERVICE to all be empty... along with literally every other env var that exists, since we've not set any 😆 .
Therefore, you can remove this line (hope the reasoning makes sense to you).